apps/trading-e2e/src/support/mocks/generate-positions.ts

```diff
@@ -0,0 +1,128 @@
+        },
+        decimalPlaces: 5,
+        tradableInstrument: {
+          instrument: {
+            id: '',
+            name: 'UNIDAI Monthly (30 Jun 2022)',
+            metadata: {
+              tags: [
+                'formerly:3C58ED2A4A6C5D7E',
+                'base:UNI',
+                'quote:DAI',
+                'class:fx/crypto',
+                'monthly',
+                'sector:defi',
+              ],
+              __typename: 'InstrumentMetadata',
+            },
+            code: 'UNIDAI.MF21',
+            product: {
+              settlementAsset: {
+                id: '6d9d35f657589e40ddfb448b7ad4a7463b66efb307527fedd2aa7df1bbd5ea61',
+                symbol: 'tDAI',
+                name: 'tDAI TEST',
+                decimals: 5,
+                __typename: 'Asset',
+              },
+              quoteName: 'DAI',
+              __typename: 'Future',
+            },
+            __typename: 'Instrument',
+          },
+          __typename: 'TradableInstrument',
+        },
+        __typename: 'Market',
+      },
+      __typename: 'Position',
+    },
+    {
+      realisedPNL: '0',
+      openVolume: '1',
+      unrealisedPNL: '-22519',
+      averageEntryPrice: '84400088',
+      market: {
+        id: '5a4b0b9e9c0629f0315ec56fcb7bd444b0c6e4da5ec7677719d502626658a376',
+        name: 'Tesla Quarterly (30 Jun 2022)',
+        data: {
+          markPrice: '84377569',
+          marketTradingMode: MarketTradingMode.Continuous,
+          __typename: 'MarketData',
+          market: {
+            __typename: 'Market',
+            id: '5a4b0b9e9c0629f0315ec56fcb7bd444b0c6e4da5ec7677719d502626658a376',
+          },
+        },
+        decimalPlaces: 5,
+        tradableInstrument: {
+          instrument: {
+            id: '',
+            name: 'Tesla Quarterly (30 Jun 2022)',
+            metadata: {
+              tags: [
+                'formerly:5A86B190C384997F',
+                'quote:EURO',
+                'ticker:TSLA',
+                'class:equities/single-stock-futures',
+                'sector:tech',
+                'listing_venue:NASDAQ',
+                'country:US',
+              ],
+              __typename: 'InstrumentMetadata',
+            },
+            code: 'TSLA.QM21',
+            product: {
+              settlementAsset: {
+                id: '8b52d4a3a4b0ffe733cddbc2b67be273816cfeb6ca4c8b339bac03ffba08e4e4',
+                symbol: 'tEURO',
+                name: 'tEURO TEST',
+                decimals: 5,
+                __typename: 'Asset',
+              },
+              quoteName: 'EURO',
+              __typename: 'Future',
+            },
+            __typename: 'Instrument',
+          },
+          __typename: 'TradableInstrument',
+        },
+        __typename: 'Market',
+      },
+      __typename: 'Position',
+    },
+  ];
+
+  const defaultResult = {
+    party: {
+      id: 'f9982447fb4128f9968f9981612c5ea85d19b62058ec2636efc812dcbbc745ca',
```

Comment:
chore (non-blocking): Can we pull this from the cypress environment? If the user is already configuring the connected key in env seems like a more sensible default

Reply:
Done